Gruntfile.coffee

```diff
@@ -1324,7 +1337,27 @@ module.exports = (grunt) ->
 					base: "<%= coreDist %>"
 					message: ((
 						if process.env.TRAVIS_TAG
-							"Variant files for the " + process.env.TRAVIS_TAG + " maintenance release"
```

Comment:
Should we just use "release" rather than "maintenance release" to cover non-maintenance releases or is there a separate message for those types of releases?